group/templates/group/groupDetail.html

```diff
@@ -82,26 +90,34 @@ <h4 class="panel-title">
         </div>
         <div id="collapseOne" class="panel-collapse collapse in">
           <div class="panel-body">
-            <table class="table table-hover">
+            <table class="table table-hover table-striped">
               <thead>
                 <tr>
-                  <th class='col-md-2'>Title</th>
-                  <th class='col-md-4'>Start Time</th>
-                  <th class='col-md-4'>End Time</th>
+                  <th>Title</th>
+                  <th>Start Time</th>
+                  <th>End Time</th>
                 </tr>
               </thead>
               <tbody>
                 {% for running_contest in rc_list %}
-                <tr>
-                  <td>{{ running_contest.name }}</td>
-                  <td>{{ running_contest.s_time }}</td>
-                  <td>{{ running_contest.e_time }}</td>
-                </tr>
+                  {% if running_contest.is_homework %}
+                    <tr style="background-color:#F1F7EA">
```

Comment:
should I use a new file to set the css or add <style> part at template to set css?